README.md

```diff
@@ -1,2 +1,69 @@
-# fire-data-processing
-Scripts to produce the fuel moisture and flammability data for the AFMS system
+##### Fire Data Processing Repository
+# ANU-WALD
+
+The Fire Data Processing repository is a collection of scripts used to produce the fuel moisture and flammability data for usage on the online interactive [Australian Flammability Monitoring system (AFMS)](http://wenfo.org/afms/#_/Fenner%3AFMC%3AC6/gfe_fire_weather/_/_/_/4/8/2017/_). This is a research initiative to provide quality spatial information on fire hazards in Australia to better understand the risk of bushfires, their severity and preparing ways to respond to them.
```

Comment:
Break paragraphs into shorter lines, or around 80 characters - to make reviewing easier and future diffs more useful.